main/src/ca/uwaterloo/flix/language/phase/CodeGen.scala

```diff
@@ -100,13 +100,14 @@ object CodeGen extends Phase[ExecutableAst.Root, ExecutableAst.Root]{
     // 3. Create Enum Type info
     val allEnums : List[(Type, (String, Type))] = root.definitions.values.flatMap(x => CodegenHelper.findEnumCases(x.exp)).toList
 
-    val enumTypeInfo: Map[(Type, String), QualName] = allEnums.map{ case (tpe, (name, subType)) =>
+    val enumTypeInfo: Map[(Type, String), (QualName, Boolean)] = allEnums.map{ case (tpe, (name, subType)) =>
```

Comment:
What is this boolean?

The type signature of this map is a bit hard to read; it goes from a pair of (Type, String) to (QualName, Boolean) - what are these components?

EDIT: I guess the boolean is whether it is singleton.